### PR TITLE
TVB-2572 Integrate Swagger with REST api

### PR DESCRIPTION
**Pull request by gabiflorea10**

### CODE REVIEW

File: framework_tvb/tvb/interfaces/rest/server/rest_api.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 #
 #
 
-from flask_restful import Api
+from flask_restplus import Api
```
Comment (liadomide) [inline framework_tvb/tvb/interfaces/rest/server/rest_api.py]: Are you certain there are only 2 imports ?
We should make sure we do not depend on both libraries

File: framework_tvb/tvb/interfaces/rest/server/run.py [hunk 2]
```diff
@@ -76,19 +76,21 @@ def initialize_flask():
...
+
+    name_space.add_resource(GetUsersResource, '/users')
+    name_space.add_resource(GetProjectsListResource, '/users/<string:username>/projects')
+    name_space.add_resource(GetDataInProjectResource, '/projects/<string:project_gid>/data')
+    name_space.add_resource(GetOperationsInProjectResource, '/projects/<string:project_gid>/operations')
+    name_space.add_resource(RetrieveDatatypeResource, '/datatypes/<string:datatype_gid>')
+    name_space.add_resource(GetOperationsForDatatypeResource, '/datatypes/<string:datatype_gid>/operations')
+    name_space.add_resource(FireSimulationResource, '/simulation/<string:project_gid>')
+    name_space.add_resource(LaunchOperationResource, '/operations/<string:project_gid>/algorithm'
+                                                         '/<string:algorithm_module>/<string:algorithm_classname>')
```
Comment (liadomide) [inline framework_tvb/tvb/interfaces/rest/server/run.py]: Is this code properly aligned ?

**Comment by liadomide** [comment]
flask_restplus should be added in setup.py also, right?

File: framework_tvb/tvb/interfaces/rest/server/rest_api.py (hunk 1 above)
Comment (gabiflorea10) [reply framework_tvb/tvb/interfaces/rest/server/rest_api.py]: Yes, there are only 2 imports.

File: framework_tvb/tvb/interfaces/rest/server/run.py (hunk 2 above)
Comment (gabiflorea10) [reply framework_tvb/tvb/interfaces/rest/server/run.py]: The problem will be fixed in a later commit.

**Comment by gabiflorea10** [comment]
> flask-restplus should be added in setup.py also, right?

yes

**Comment by gabiflorea10** [comment]
> Changes look ok. Pls just answer my concerns in the comments. If everything confirmed, then we can merge

I suggest to not merge yet, because some methods are not working right know.
After fixing these methods, the code could be merged.

**Pull request merged**